identity/backend/src/main/java/io/camunda/identity/controller/UserController.java

```diff
@@ -1,57 +0,0 @@
-/*
```

Comment:
I have removed that as it will change and move to gateway-rest to be part of the unified api in a separate PR once this one here is merged.